src/main/resources/application.yml

```
@@ -0,0 +1,57 @@
+logging:
+  level.root: WARN
+  level.no: ${LOG_LEVEL:DEBUG}
+  level.org.springframework: WARN
+  level.org.springframework.web: WARN
+  level.org.apache.jena: ERROR
+spring:
+  rabbitmq:
+    host: rabbitmq
+    port: 5672
+    username: ${RABBIT_USERNAME}
+    password: ${RABBIT_PASSWORD}
+server:
+  port: 8080
+application:
+  uri:
+    datasets: http://fdk-dataset-harvester:8080/catalogs
+    dataservices: http://fdk-dataservice-harvester:8080/catalogs
+    concepts: http://fdk-concept-harvester:8080/concepts
+    informationmodels: http://fdk-informationmodel-harvester:8080/catalogs
+    events: http://fdk-event-harvester:8080/events
+    organizations: http://organization-catalogue:8080/organizations
+    publicservices: http://fdk-public-service-harvester:8080/public-services
+    sparqlservice: http://fdk-sparql-service:8080
```

Jeg ser ikke behov å lage nøsting her:
```suggestion
    datasets-uri: http://fdk-dataset-harvester:8080/catalogs
    data-services-uri: http://fdk-dataservice-harvester:8080/catalogs
    concepts-uri: http://fdk-concept-harvester:8080/concepts
    information-models-uri: http://fdk-informationmodel-harvester:8080/catalogs
    events-uri: http://fdk-event-harvester:8080/events
    organizations-uri: http://organization-catalogue:8080/organizations
    public-services-uri: http://fdk-public-service-harvester:8080/public-services
    sparql-service-uri: http://fdk-sparql-service:8080
```

---

Planen var vel å ha flere properties-filer, men kom tydeligvis ikke så langt.

---

Nå er det mer behov for nøsting: ddbfbbf